libraries/rodeos/include/b1/rodeos/callbacks/coverage.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <b1/rodeos/callbacks/vm_types.hpp>
+#include <eosio/coverage.hpp>
+
+namespace b1::rodeos {
+
+class coverage_maps {
```

Comment:
Instead of duplicating this code, make it a template with unique values:

```
template<eosio::name Unique>
class coverage_maps {
```
Then use like:

```
coverage_maps<"rodoes"_n>
coverage_maps<"nodeos"_n>
```

Reply:
Done, although eosio::name was causing me some trouble with includes so I'm just using uint64_t for the template param, in the code I do use a eosio::name constant.